Lib/axisregistry/data/element_grid.textproto

```diff
@@ -7,3 +7,7 @@ max_value: 2.0
 precision: -1
 description:
   "Controls how many elements are used per one grid unit."
+fallback {
+  name: "Single element"
```

Comment:
The CI is failing on the space in the name here

Reply:
That doesn't seem right, but fixed by changing to Default